distributions/demo-google-deployment/resources/log4j.properties

```diff
@@ -0,0 +1,5 @@
+log4j.rootLogger=DEBUG, A1
```

Comment:
License Header plus documentation on purpose of this file.

Reply:
Done